code/modules/clothing/under/f13.dm

```diff
@@ -1512,3 +1512,10 @@
 	item_state = "scoutclothes"
 	item_color = "scoutclothes"
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 30, "fire" = 20, "acid" = 50)
+	
+/obj/item/clothing/under/f13/densuit
+	name = "the den outfit"
+	desc = "A dark grey, and finely pressed suit, complete with kneepads and a suspiciously golden silk shirt, only the best."
+	icon_state = "den_suit"
+	item_state = "den_suit"
+	item_color = "den_suit"
```

Comment:
```suggestion
```

Reply:
`item_color` isn't required, it certainly won't break anything, but is just unnecessary.